packages/core/src/Point.ts

```diff
@@ -123,17 +123,20 @@ export class Point {
   }
 
   /**
-   * Sets point time. A string or number value can be used
-   * to carry an int64 value of a precision that depends
-   * on WriteApi, nanoseconds by default. An undefined value
-   * generates a local timestamp using the client's clock.
-   * An empty string can be used to let the server assign
-   * the timestamp.
+   * Sets point timestamp. Timestamp can be specified as a Date, number, string,
+   * bigint or undefined value. A number or a string (base-10) value
+   * represents time as a count of time units since epoch. The time unit
+   * is specified in a WriteApi precision, nanosecond by default.
+   * An undefined value instructs to assign a local timestamp using
+   * the client's clock. An empty string can be used to let the server assign
+   * the timestamp. Bigint (or any other value type) can be also passed, its
+   * toString() value is used. Note that InfluxDB requires the timestamp to fit into
+   * int64 data type.
```

Comment:
This is much better than my edit, thanks!

Reply:
I'll have to repair it a bit, bigint would actually fail in the current implementation :)